test/regtest.py

```diff
@@ -69,29 +73,25 @@ def metadata(file):
         m['expect'] = match.group(1).strip()
 
   if not m['skip'] and not 'expect' in m:
-    print red("WARNING: @expect MISSING IN %s" % file)
+    logging.warning("@expect MISSING IN %s" % file)
```

Comment:
This call to `logging.warning(..)` is not multi-process-safe — see discussion below.